translations/en_GB/atrium-translations-en_GB-common/src/main/kotlin/ch/tutteli/atrium/translations/DescriptionBasic.kt

```diff
@@ -28,5 +28,6 @@ enum class DescriptionBasic(override val value: String) : StringBasedTranslatabl
     HAS("has"),
     HAS_NOT("has not"),
     WAS("was"),
-    NONE("none")
+    NONE("none"),
+    CONTAINS_DUPLICATES("contains duplicates")
```

Comment:
in case you still use this, then move to DescriptionIterableAssertions

Reply:
moved